docs/topics/land/land.md

```diff
@@ -0,0 +1,369 @@
+}
+
+class Land{
+    + address metaTransactionContract
+    + address admin
+    + name(...)
+    + symbol(...)
+    + tokenURI(...)
+    + supportsInterface(...)
+}
+
+interface ERC721Events{}
+
+class SuperOperator{
+    {field} # _superOperators: mapping (address => bool)
+    + setSuperOperator(superOperator, enabled)
+    + isSuperOperator(who)
+}
+
+class MetaTransactionReceiver{
+    {field} # _metaTransactionContracts: mapping (address => bool)
+    + isMetaTransactionProcessor(who)
+}
+
+
+
+
+ERC721Events <|-- ERC721BaseToken
+SuperOperator <|-- ERC721BaseToken
+MetaTransactionReceiver <|-- ERC721BaseToken
+ERC721BaseToken <|-- LandBaseToken
+LandBaseToken <|-- Land
+```
+
+## Processes
+
+#### Illustrating how quads work:
+
+_This is not a user guide, users don't see quads, from their point of view they are only dealing with 1x1 lands_
+
+<!-- you say that but users can call transferQuad with x, y, size so they have to be aware of quads.. -->
+
+Let's imagine we're minting a 6x6 quad for user A, in the coordinates 0,0. We'll mint a single token, with an quad id calculated by:
+
+```
+    uint256 internal constant LAYER_6x6 =
+    0x0200000000000000000000000000000000000000000000000000000000000000;
+    uint256 quadId;
+        uint256 id = x + y * GRID_SIZE;
+        ...
+        .. if (size == 6) {
+            quadId = LAYER_6x6 + id;
+```
+
+There won't be 36 tokens minted, just one, with an id that allows us to access all it's internal quads.
+
+![](../../img/quadIMG1.png)
+
+User A can transfer his 6x6 quad to user B if he wants to, calling the transferQuad function.
+
+![](../../img/quadIMG2.png)
+
+In this 6x6 example, the quad can also be separated in 3x3 ones, and each 3x3 can be separated into 1x1 ones. The next image shows what would happens if a 3x3 quad would be transfered to user B.
+
+![](../../img/quadIMG3.png)
+
+It is also possible to transfer 1x1 quads.
+
+![](../../img/quadIMG4.png)
+
+Now, if we try to transfer the 3x3 from which we took one of the 1x1 quads, we'll get an error "not owner"
+
+![](../../img/quadIMG5.png)
+
+If the user A wants to send the rest of these lands, he would need to send the 1x1 quads one by one.
+![](../../img/quadIMG6.png)
+
+### Mint a quad
+
+In order to mint multiple lands without the need of multiple transactions, lands are minted in quads. Only the minter can call the `mintQuad` function, upon it's call, it is necessary to inform
+
+- `to`: who’s going to receive the land tokens,
+- the `size` of the lands being minted,
+- `x` and `y`
+- and `data`: extra data to pass to the transfer.
+
+The coordinates `x` and `y`, are the coordinates of the top left point of a quad, they must be multiples of the quad size and should respect the bounds of the land grid.
+
+The function will then calculate a quadId, it starts with a base x + y \* GRID_SIZE;
+The rest of the calculation depends on the size of the Lands being minted.
+
+!!! quadId calculation
+
```

Comment:
remove the break line, that breaks the admonition
and it has to be:
```!!! example quadId calculation```

https://squidfunk.github.io/mkdocs-material/reference/admonitions/#inline-blocks